appveyor-install-deps.cmd

```diff
@@ -0,0 +1,6 @@
+"%sdkverpath%" -q -version:"%sdkver%"
+call setenv /x64
+
+pip install --cache-dir c:/temp numpy
+pip install --cache-dir c:/temp distribute
+pip install --cache-dir c:/temp enable
```

Comment:
I think you should install enable from git here, similar to what is done for Travis.